test/onnx/exporter/test_small_models_e2e.py

```diff
@@ -752,8 +770,12 @@ def forward(self, query, key, value):
         key = torch.rand(32, 8, 128, 64, dtype=torch.float16)
         value = torch.rand(32, 8, 128, 64, dtype=torch.float16)
 
-        onnx_program = self.export(Model(), (query, key, value), opset_version=23)
-        onnx_testing.assert_onnx_program(onnx_program, atol=1e-3, rtol=1)
+        onnx_program = self.export(Model(), (query, key, value), opset_version=23, optimize=True)
+        proto = onnx_program.model_proto
+        self.assertEqual(["Attention"], [n.op_type for n in onnx_program.model.graph])
+        # onnxruntime inlines any op defined as a function and without any implemented kernel
+        if self.has_onnxruntime_opset_23():
```

Comment:
```suggestion
        if has_onnxruntime_opset_23():
```

also this helper doesn't need to be in self because it doesn't use self states